routing/router_test.go

```diff
@@ -1957,7 +1957,7 @@ func TestPruneChannelGraphStaleEdges(t *testing.T) {
 	t.Parallel()
 
 	freshTimestamp := time.Now()
-	staleTimestamp := time.Time{}
+	staleTimestamp := time.Unix(0, 0)
```

Comment:
reason for this change?

Reply:
@cfromknecht `time.Time{}.Unix()` results in a negative number. 
While querying boltdb in ChanUpdatesInHorizon (https://github.com/lightningnetwork/lnd/blob/master/channeldb/graph.go#L1380) there is a conversion to uint64 which results in a very large number serialized to `startTimeBytes`
I think this is OK and only the test should be updated to better reflect the reality.

Reply:
Very interesting, TIL!